### PR TITLE
Upgrade patternfly packages

### PR DESCRIPTION
**Pull request by pnaik1**
Closes: https://issues.redhat.com/browse/RHOAIENG-3847

## Description
Upgraded the patternfly packages to 5.3.2 (https://www.patternfly.org/get-started/release-highlights/#patternfly-5.3.2)

## How Has This Been Tested?
This was tested manually 
Manually ensure UI components renders correctly

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -54,18 +54,18 @@
   "dependencies": {
     "@openshift/dynamic-plugin-sdk": "^4.0.0",
     "@openshift/dynamic-plugin-sdk-utils": "^4.0.1",
-    "@patternfly/patternfly": "^5.2.1",
+    "@patternfly/patternfly": "^5.3.1",
     "@patternfly/quickstarts": "^5.1.0",
```
Comment (dpanshug) [inline frontend/package.json:58]: Upgrade quickstart to 5.3.0 https://www.npmjs.com/package/@patternfly/quickstarts

File: frontend/package.json [hunk 2]
```diff
@@ -54,18 +54,18 @@
...
+    "@patternfly/react-icons": "^5.3.2",
     "@patternfly/react-log-viewer": "^5.2.0",
-    "@patternfly/react-styles": "^5.2.1",
-    "@patternfly/react-table": "^5.2.1",
-    "@patternfly/react-tokens": "^5.2.1",
+    "@patternfly/react-styles": "^5.3.1",
+    "@patternfly/react-table": "^5.3.3",
+    "@patternfly/react-tokens": "^5.3.1",
     "@patternfly/react-topology": "^5.4.0-prerelease.6",
     "@patternfly/react-virtualized-extension": "^5.0.0",
```
Comment (dpanshug) [inline frontend/package.json:70]: Upgrade this to 5.1.0 https://npmjs.com/package/@patternfly/react-virtualized-extension

**Review by dpanshug — CHANGES_REQUESTED** [review]
Pls reconfirm the suggested changes from patternfly.org

File: frontend/package.json (hunk 2 above)
Comment (pnaik1) [reply frontend/package.json:70]: @dpanshug I referred this link  https://www.patternfly.org/get-started/release-highlights , I see react-virtualized-extension promoted package version is still `5.0.0`
cc @christianvogt

Comment (dpanshug) [reply frontend/package.json:70]: Maybe it was not mentioned in release highlights but the latest according to [github release](https://github.com/patternfly/react-virtualized-extension/releases/tag/v5.1.0) is 5.1.0

**Review by jeff-phillips-18 — APPROVED** [review]
/lgtm

**Pull request merged**